app/index.tsx

```diff
@@ -59,6 +59,7 @@ const Index = () => {
   return (
     <View style={styles.container}>
       <TabView
+        renderTabBar={(props) => <TabBar {...props} />}
```

Comment:
@ryanlanciaux  The issue starts here: when a custom TabBar is provided and unistyles is then used within that custom TabBar.